Jenkinsfile

```diff
@@ -46,5 +46,47 @@ pipeline {
         }
       }
     }
+    stage('Deploy') {
+      when {
+        branch 'master'
+        beforeAgent true
+      }
+      agent {
+        dockerfile {
+          additionalBuildArgs '--build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g)'
+          reuseNode true
+        }
+      }
```

Comment:
@shd101wyy these lines shouldn't exist, because we're already in a a docker container (from the top of the Jenkinsfile).